BookReaderDemo/IADemoBr.js

```diff
@@ -18,6 +18,9 @@ if (openFullImmersionTheater) {
   iaBookReader.fullscreen = openFullImmersionTheater;
 }
 
+const modal = document.querySelector('modal-manager');
```

Comment:
**question**: Why use var here?
why not
```
iaBookReader.modal = document.querySelector('modal-manager');
```
?

Reply:
bc don't really need to pin to class instance